operator/v1/types_network.go

```diff
@@ -136,23 +136,26 @@ const (
 // NetworkMigration represents the cluster network configuration.
 // +openshift:validation:FeatureGateAwareXValidation:featureGate=NetworkLiveMigration,rule="!has(self.mtu) || !has(self.networkType) || self.networkType == \"\" || has(self.mode) && self.mode == 'Live'",message="networkType migration in mode other than 'Live' may not be configured at the same time as mtu migration"
 type NetworkMigration struct {
-	// networkType is the target type of network migration. Set this to the
-	// target network type to allow changing the default network. If unset, the
-	// operation of changing cluster default network plugin will be rejected.
-	// The supported values are OpenShiftSDN, OVNKubernetes
-	// +optional
-	NetworkType string `json:"networkType,omitempty"`
-
 	// mtu contains the MTU migration configuration. Set this to allow changing
 	// the MTU values for the default network. If unset, the operation of
 	// changing the MTU for the default network will be rejected.
 	// +optional
 	MTU *MTUMigration `json:"mtu,omitempty"`
 
+	// networkType is the target type of network migration. Set this to the
+	// target network type to allow changing the default network. If unset, the
+	// operation of changing cluster default network plugin will be rejected.
+	// Current versions of OpenShift do not support networkType migration and
+	// will reject any configuration where this is set.
+	// +optional
+	NetworkType string `json:"networkType,omitempty"`
```

Comment:
Could you help me understand why are we not deprecating networkType migration?

Reply:
Hm... I guess you're right. We have no plans of ever supporting that again.